geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/KryoLazyAggregatingIterator.scala

```diff
@@ -83,14 +85,7 @@ abstract class KryoLazyAggregatingIterator[T <: AnyRef { def isEmpty: Boolean; d
     val transform = Option(jOptions.get(TRANSFORM_DEFINITIONS_OPT))
     val transformSchema = Option(jOptions.get(TRANSFORM_SCHEMA_OPT))
     for { t <- transform; ts <- transformSchema } {
-      reusableSf.setTransforms(t, IteratorCache.sft(ts))
-
-      reusableTransformSf = if (index.serializedWithId) {
-        IteratorCache.serializer(ts, SerializationOptions.none).getReusableFeature
-      } else {
-        IteratorCache.serializer(ts, SerializationOptions.withoutId).getReusableFeature
-      }
-
+      reusableTransformSf = TransformSimpleFeature(IteratorCache.sft(spec), IteratorCache.sft(ts), t)
```

Comment:
[minor] can use the existing `sft` instead of retrieving from cache again with `IteratorCache.sft(spec)`